Branch-SDK/Branch-SDK/BNCPreferenceHelper.h

```diff
@@ -31,8 +31,9 @@
 @property (assign, nonatomic) BOOL explicitlyRequestedReferrable;
 @property (assign, nonatomic) BOOL isReferrable;
 @property (assign, nonatomic) BOOL isDebug;
-@property (assign, nonatomic) BOOL isContinuingUserActivity;
+@property (assign, nonatomic) BOOL shouldWaitForInit;
```

Comment:
Are you sure it's in the wrong place? I'm not sure this change is right. I'll review when I'm in front of a computer, likely tomorrow. Either way, this is my fault for a poorly named variable